documentation/dev/introduction.rst

```diff
@@ -152,22 +152,37 @@ You can run automated tests with:
    $ make test
 
 
-which behind the curtains installs dependencies and calls pytest.
+which behind the curtains installs dependencies and calls ``pytest``.
 
-A coverage report can be created like this:
+However, a test postgres database is needed to run these tests. If you have postgres, here is the short version on how to add the test database:
 
 .. code-block:: bash
 
-   $ pytest --cov=flexmeasures --cov-config .coveragerc
+    make clean-db db_name=flexmeasures_test db_user=flexmeasures_test
+    # the password for the db user is "flexmeasures_test"
+
+.. note:: The section :ref:`host-data` has more details on using postgres for FlexMeasures.
+
+Alternatively, if you don't feel like installing postgres for the time being, here is a docker command:
+
+.. code-block:: docker
```

Comment:
Still bash I guess. Or does this apply special formatting in sphinx?

Reply:
Correct. I believe Docker formatting would be for Dockerfiles.